### ISSUE TITLE
move forward and elevation

### ISSUE THREAD

**Issue by vegarringdal** [yomotsu/camera-controls#367]
### Is your feature request related to a problem? Please describe.

yes, its hard to move forward and change elevation at the same time

In my app I use WASD to move like first person view, but i also allow user to use Q and E to change elevation.
Problem is when user holds W and Q or E in at the same time. There is no good way to set forward and elevation change at the same time

### Describe the solution you'd like

I extended local copy of camera-controls with this function.
Have both Z and Y since some have Z as UP and not Y.

```ts
forwardElevation(distance, enableTransition = false, offsetZ = 0, offsetY = 0) {
	_v3A.setFromMatrixColumn(this._camera.matrix, 0);
	_v3A.crossVectors(this._camera.up, _v3A);
	_v3A.multiplyScalar(distance);
	const to = _v3B.copy(this._targetEnd).add(_v3A);
	return this.moveTo(to.x, to.y + offsetY , to.z + offsetZ , enableTransition);
}
```

### Additional context

ref https://github.com/yomotsu/camera-controls/issues/332

**Comment by yomotsu** [comment]
Thanks for your feature request.
How about only focusing on elevation in this function?

Like:
```
elevate( height: number, enableTransition: boolean = false ) {

  return this.moveTo( this._targetEnd.x, this._targetEnd.y + height, this._targetEnd.z, enableTransition );

}
```

then if you wish to combine with moving forward:

```
cameraControls.forward( 2, true );
cameraControls.elevate( 1, true );
```

**Comment by VegarRingdalAibel** [comment]
Hi
Yes sounds like a good idea.

**Comment by yomotsu** [comment]
Thanks for understanding.
Are you interested in becoming a contributor (with making a PR)?
Or shall I do that later?

**Comment by vegarringdal** [comment]
Its ok, it will be faster if you do it 😁
Maybe Ill try next time. 👍

**Comment by yomotsu** [comment]
elevate is now available! closing this issue.

**Comment by vegarringdal** [comment]
thank you 👍

**Comment by VegarRingdalAibel** [comment]
I just updated from version 2.0.1
Looks have multitouch zoom been removed/changed?

**Comment by yomotsu** [comment]
@vegarringdal thanks for your report. I just fixed the problem and 2.3.1 was just published.
BTW, it would be helpful if you could open a new issue rather than using a closed issue when you find a bug.

**Comment by vegarringdal** [comment]
Sorry, will open new issue next, sorry for being lazy 😁
Thanks for fixing the issue, will give it try tomorrow.
